src/Microsoft.ML.Tokenizers/Model/EnglishRoberta.cs

```diff
@@ -136,12 +137,12 @@ public EnglishRoberta(Stream vocabularyStream, Stream mergeStream, Stream highes
             skipSpecialTokens && id < 0 ? null : _vocabReverse.TryGetValue(id, out var value) ? value : null;
```

Comment:
For constructor maybe consider adding a constructor that takes in a `Dictionary<string, int> vocab, (string, string)[] merge)` as well? Because in some tokenizer config, the merge.txt might contain items that include a newline, like below

```
\", \r
...
```

And that will break both constructors in the current EnglishRoberta or BPE

Reply:
Talked offline with @LittleLittleCloud and we decided instead we can think in supporting a full model file like [tokenizer.json](https://huggingface.co/microsoft/phi-2/raw/main/tokenizer.json) in future.